### PR TITLE
Don't force args for install

### PR DESCRIPTION
**Pull request by conico974**
For some libs it seems that the default args that we used doesn't work, we can make all of them optional

### CODE REVIEW

File: packages/open-next/src/build/installDeps.ts [hunk 1]
```diff
@@ -23,11 +23,18 @@ export function installDependencies(
...
+      : "";
+    const targetOption = installOptions.nodeVersion
+      ? `--target=${installOptions.nodeVersion}`
+      : "";
+    const libcOption = installOptions.libc
+      ? `--libc=${installOptions.libc}`
+      : "";
+
+    const additionalArgs = installOptions.additionalArgs ?? "";
+    const installCommand = `npm install --platform=linux ${archOption} ${targetOption} ${libcOption} ${additionalArgs} ${installOptions.packages.join(" ")}`;
```
Comment (khuezy) [inline packages/open-next/src/build/installDeps.ts:37]: what if the platform is `Windows`?

Comment (conico974) [reply packages/open-next/src/build/installDeps.ts:37]: People still use windows as a server ? :joy: 
That's only for the server. Honestly there is probably dozens of other issue trying to run OpenNext on windows.

Comment (khuezy) [reply packages/open-next/src/build/installDeps.ts:37]: I read someone was insterested in running ON on Azure 🤷 
We'll cross that bridge when we get there.

**Review by khuezy — APPROVED** [review]
🚢 it

**Pull request merged**